src/UserSpaceInstrumentation/UserSpaceInstrumentationTestLib.h

```diff
@@ -5,19 +5,20 @@
 #ifndef USER_SPACE_INSTRUMENTATION_TEST_LIB_H_
 #define USER_SPACE_INSTRUMENTATION_TEST_LIB_H_
 
-#include <string_view>
-
 // Library functions for some trivial logging. This library is merely used in tests: The test
 // injects a binary produced by this code into its child.
 
 // Call first to initialize the library.
-void InitTestLib();
+extern "C" void InitTestLib();
 
 // Log a string into a temporary file.
-void UseTestLib(std::string_view s);
+extern "C" void UseTestLib(const char* s);
 
 // Call to end using the library. Prints the entire log to standard output and removes the
 // temporary log file.
-void CloseTestLib();
+extern "C" void CloseTestLib();
+
+// Returns 42.
+extern "C" int TrivialFunction();
```

Comment:
Is this function used?

Reply:
It is needed in a follow up PR ... bear with me.

Reply:
Sounds good.